packages/store/ts/config/v2/table.test.ts

```diff
@@ -177,7 +32,7 @@ describe("resolveTableConfig", () => {
       primaryKey: ["key"],
     } as const;
 
-    attest<typeof expected>(table);
+    attest<typeof expected>(table).snap(expected);
```

Comment:
```suggestion
    attest<typeof expected>(table).equals(expected);
```
wasn't sure if `.snap` was intended here